### PR TITLE
Mange team/domain matching docs

### PR DESCRIPTION
**Pull request by ngrayluna**
## Description

Refactored Access management section:
1. Consolidated and update content in [Manage users](https://docs.wandb.ai/guides/hosting/iam/manage-users) and added said content to a new page called [Managed your organization](https://bedb66be.docodile.pages.dev/guides/hosting/iam/manage-organization).
2. Added a landing page for [Access management](https://bedb66be.docodile.pages.dev/guides/hosting/iam/access-management-intro).

Pages to review: 
* [Manage your organization](https://bedb66be.docodile.pages.dev/guides/hosting/iam/manage-organization)
   * Domain matching content can be found here: https://bedb66be.docodile.pages.dev/guides/hosting/iam/manage-organization#automatically-add-new-users-to-a-team
* Access management landing page [Do not review yet]

To do: 
* Add intro blurb on Access management page. 
* Possibly add more images to Manage your organization to break up the wall of text.
* Follow up PR for these subsections (See commented out content at the bottom of `manage-organization.md` file):
   * Manage team storage
   * Create and assign custom roles
   * Privacy  

## Future URL

Assuming we don't change the page name, the future URL will be:

`https://docs.wandb.ai/guides/hosting/iam/manage-organization`

## Ticket

https://wandb.atlassian.net/browse/DOCS-962

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/hosting/iam/manage-organization.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+:::
+
+If you are looking to simplify user management in your organization, refer to [Automate user and team management](./automate_iam.md).
+
+
+<!-- W&B assigns an Admin role to new users within an organization by default.  -->
+
+## Change the name of your organization
+
+1. Navigate to the organization's dashboard at https://wandb.ai/home.
```
Comment (cassface) [inline docs/guides/hosting/iam/manage-organization.md]: I believe this is incorrect, for both SaaS and server... 

There's no org display name in server today.

In SaaS the org display name is editable via https://wandb.ai/account-settings/wandb/settings for customers this pattern would be https://wandb.ai/account-settings/{Org Name}/settings

File: docs/guides/hosting/iam/manage-organization.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+2. Choose the User menu in the upper right corner of the organization dashboard. From the dropdown, Choose **Settings**.
+3. Within the **Settings** tab, select **General**.
+4. Select the **Change name** button.
+5. Within the modal that appears, provide a new name for your organization and select the **Save name** button.
+
+## Add and manage users
+
+Use your organization's dashboard to invite users, assign or update a user's role, remove users from your organization, assign the billing administrator, and more. Your organization's dashboard is located at: https://wandb.ai/home.
```
Comment (cassface) [inline docs/guides/hosting/iam/manage-organization.md]: our organization's dashboard is located at: https://wandb.ai/home.

This is incorrect. see above comment

File: docs/guides/hosting/iam/manage-organization.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+### Invite a user
+
+1. Navigate to your organization's dashboard at https://wandb.ai/home.
```
Comment (cassface) [inline docs/guides/hosting/iam/manage-organization.md]: This needs to be corrected

File: docs/guides/hosting/iam/manage-organization.md (hunk 1 above)
Comment (ngrayluna) [reply docs/guides/hosting/iam/manage-organization.md]: @cassface What's the official name for `https://wandb.ai/home` ?  

In the instructions, this is step 1 but then in the subsequent steps I describe to the go to the Settings page. etc

File: docs/guides/hosting/iam/manage-organization.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+:::tip enable SSO for authentication
+W&B strongly recommends and encourages that users authenticate to an organization using Single Sign-On (SSO). 
+
+To learn more about how to setup SSO with Dedicated cloud or Self-managed instances, refer to [SSO with OIDC](./sso.md) or [SSO with LDAP](./ldap.md).
+
+Reach out to your W&B for further assistance.
+:::
+
+### Auto provision users
```
Comment (ngrayluna) [inline docs/guides/hosting/iam/manage-organization.md]: @cassface Is auto provisioning applicable for Multi tenant? Or only Dedicated and/or Self managed?

File: docs/guides/hosting/iam/manage-organization.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
+---
+displayed_sidebar: default
+---
+
+# Mange your organization
```
Comment (cassface) [inline docs/guides/hosting/iam/manage-organization.md]:
``` suggestion
# Manage your organization
```

**Pull request merged**